src/editor/auto-entities-filter-editor.ts

```diff
@@ -144,6 +146,11 @@ class AutoEntitiesFilterEditor extends LitElement {
       );
       (fold as any).expanded = true;
     });
+
```

Comment:
No.

I'm done with backwards compatibility. That way lies madness.

The right way is to define the minimum Home Assistant version in hacs.json
https://github.com/thomasloven/lovelace-auto-entities/blob/master/hacs.json#L4

Reply:
Will update.